content/docs/extensions/pgvector.md

```diff
@@ -73,14 +73,31 @@ For more information about querying vectors, refer to the [pgvector README](http
 
 Using an index on the vector column can improve query performance with a minor cost in recall.
 
-You can add an index for each distance function you want to use. For example, the following query adds an index to the `embedding` column for the L2 distance distance function:
+You can add an index for each distance function you want to use. For example, the following query adds an index to the `embedding` column for the L2 distance function:
 
 ```sql
 CREATE INDEX ON items USING ivfflat (embedding vector_l2_ops) WITH (lists = 100);
 ```
 
 For additional indexing guidance and examples, see [Indexing](https://github.com/pgvector/pgvector/tree/8bf360ed84bfdeba9caa19e9f193fd9ad8dd9e73#indexing), in the _pgvector README_.
 
+<Admonition type="note">
+If you encounter a `not enough memory` error while attempting to create an index, you can increase the `maintenance_work_mem` setting to the required amount of memory using a `SET` or `ALTER DATABASE` statement. The default `maintenance_work_mem` setting is 64MB. The `SET` statement changes the value for the current session. `ALTER DATABASE` updates the session default.
```

Comment:
I created a compute with 7 CUs, and the maintenance_work_mem setting for that compute was 481313kBs. I've dropped mention of the 64MB default, since it might be confusing for users with a bigger compute. I mentioned instead that the default depends on compute size.